File: mobile/src/main/java/com/yalantis/watchface/view/configuration/ConfigurationMvpView.java
package com.yalantis.watchface.view.configuration;

import android.content.Intent;

import com.yalantis.watchface.view.MvpView;

public interface ConfigurationMvpView extends MvpView {

    void startFileChooser(Intent intent, int requestCode);

    void showStatusMessage(String message);

}

File: mobile/src/main/java/com/yalantis/watchface/view/configuration/ConfigurationMvpView.kt
package com.yalantis.watchface.view.configuration

import android.content.Intent
import com.yalantis.watchface.view.MvpView

interface ConfigurationMvpView : MvpView {

    fun startFileChooser(intent: Intent, requestCode: Int)

    fun showStatusMessage(message: String)
}
